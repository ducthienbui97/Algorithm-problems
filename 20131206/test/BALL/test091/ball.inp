9 480350 458333 126338 957230
72005 897229 285655 665836 499394 326014 739826 250391 129795 178046 649799 384359 468647 669305 463616 470740 792652 442846 168760 461910 327200 388635 160144 934986 282625 434443 837955 846178 359719 104249 338430 251404 990718 154089 539951 294383 235595 236243 320812 118180 25095 594779 106668 442943 368235 839599 571048 305461 857861 843425 451375 881979 204307 429574 274018 696391 703620 711996 759026 946606 216504 923032 849636 711477 834221 357035 221623 792389 372200 482590 88446 442834 462570 530685 611057 780852 705644 642825 433682 509444 481770 
951586 702563 148296 309854 79340 269151 250696 517697 354813 467794 334565 143692 947486 27973 365087 921795 468684 276130 914554 802585 543244 970306 765071 205879 269539 163937 309844 505699 978076 454648 808387 52223 54910 155973 795751 504044 558095 737065 365591 687299 643207 697746 680599 251952 162884 736561 129810 931024 810916 321108 190793 180768 116939 206658 210839 893392 668587 169953 586128 612837 811870 454352 209177 725165 442529 490193 6071 462718 191812 536343 184264 157805 99263 945637 45422 145142 28154 624242 284979 115055 588051 
270938 551642 85523 593543 519954 323287 283152 595904 828513 980036 444659 65603 797054 470868 596756 930490 323760 461736 893902 613750 71369 860446 582780 438177 127537 590676 406744 52499 355571 196117 109455 742615 154947 904364 230093 363633 877980 214849 139718 767359 614491 933147 515574 958410 594636 832629 462831 336241 641419 766261 118941 384408 241615 229292 208321 597735 603120 93977 739765 202761 972778 857248 575808 471633 550243 162478 494642 336465 66077 810393 909751 394674 283104 430936 425959 190726 257085 303036 513653 646352 993253 
895105 446857 19356 964774 990163 718234 610244 785161 744977 262143 284784 561228 168811 581344 402267 664694 351115 114652 565651 285492 7507 937013 341428 397118 426209 640279 635750 725065 473441 332775 484424 655693 102410 338434 780961 245174 193095 176714 913108 955540 14130 830032 187338 22021 374032 265164 856691 878225 224854 973504 389883 767843 122357 390621 394696 700707 192237 473202 709274 746708 250381 371075 764299 497224 922389 854605 578885 935697 616303 520559 853523 531706 854338 131513 660112 221568 453035 902400 633785 470655 49478 771232 735571 819950 204013 904843 535593 21957 23953 870480 270958 841433 619220 365812 528770 104753 551657 946907 10707 498670 121106 164643 282409 446013 328722 452900 446495 282071 359704 59247 368270 716393 129386 523258 868868 279708 219227 443433 900857 64807 350056 934678 237158 652229 427063 358713 33439 453712 544911 886895 325526 507936 591796 568531 885627 421875 383126 608291 986137 585148 687552 709479 482138 2180 9746 254975 796423 13398 783209 60428 165781 331484 460897 575328 796103 774479 23184 851515 443042 225047 854237 898980 343336 632466 853838 254801 801585 821621 724882 615220 52878 699646 451944 111124 894535 945405 569446 643722 200266 812140 148289 168429 456251 442020 823930 780202 780182 83688 556973 481950 695078 484183 148043 464147 164603 968426 854534 947002 825711 307536 100290 70917 241540 6789 177390 591913 570498 526685 668340 721391 67593 216159 271890 246511 257630 842612 60645 114081 924826 323571 827592 375403 245389 215973 842021 929377 507732 373287 101360 632875 346416 652166 459177 858220 385795 819802 775369 119035 787821 627259 930957 774288 409400 132757 292895 862588 718280 272696 653689 627446 269214 937218 516746 507294 43863 934731 942901 945890 485271 73442 346174 733138 63816 179564 64674 280095 812022 73334 858860 826996 810085 375239 204733 574698 126307 72439 779157 818995 171757 219257 131795 841817 243119 449651 233892 913917 423604 704206 327392 332542 596259 482821 208364 878517 965757 453034 248574 938197 360555 49902 244291 806775 777746 876732 441921 518324 77744 761001 820668 39609 976978 702801 977974 251035 419060 257418 640164 78085 289110 841523 600325 288938 83307 886838 710279 245454 151372 260471 616711 699595 290507 7307 341158 229755 368931 494371 252887 681361 273453 423256 114718 795542 114354 62452 502455 338791 421463 568351 150969 299795 784733 958683 432998 272788 250480 49392 252238 360341 383909 876365 329001 970121 134572 19490 816690 285457 550700 990244 354403 380665 2910 646189 720367 389740 38951 18659 584122 744584 223859 746542 713361 780271 229981 916375 858629 850389 375798 333066 207603 284489 491518 792051 766563 208564 504786 743657 936050 614587 195297 773459 679614 271995 580911 491629 856791 818691 853269 314625 915924 301691 816153 265999 533455 243597 355521 321250 45797 240325 777981 834556 3880 869840 951352 109461 533590 697136 458150 25066 965905 59082 771191 61868 455752 256044 554865 47630 327144 67787 302189 671903 280572 570663 870750 659271 285464 775349 750871 528541 526872 90981 415155 452652 472908 652876 590813 930012 214765 178476 314614 950848 887325 149648 606819 997613 468168 877671 394926 612388 43520 191567 101176 207488 858037 849005 380925 90164 325953 789006 207745 928364 549750 845991 636015 752716 186426 790042 878084 274039 626912 841777 429366 524701 319589 320133 128762 433500 294745 718210 89057 957279 924611 718706 43527 23284 723401 847243 54568 392110 764233 170175 214374 806980 133657 833644 749288 151247 76557 526149 737659 921648 845824 627031 289676 906887 661018 48565 921564 112127 41847 262759 947958 789671 496331 959537 489954 266384 203434 833963 793344 649846 316324 643314 330351 396449 447150 708072 342519 24022 267965 34898 705324 935682 536747 969060 264350 694316 77593 473894 248887 122026 748917 771706 566696 411294 455375 566188 990904 439678 867125 829577 80323 373009 436101 105336 80643 389174 171314 5870 729800 438846 619853 192985 480805 124458 718840 505686 74964 502267 428792 608596 370453 38187 416257 138990 695154 955476 744453 677113 179994 89059 328009 784395 484688 344632 619624 491723 903093 757667 858146 174336 329768 365518 141537 136197 700688 241902 962564 498418 214787 570489 909093 619813 707588 761869 406383 380088 129841 755974 281798 130235 410926 937693 691863 726478 875145 606574 381150 932372 825345 197895 93732 50710 144482 767430 319371 413383 38262 107622 636054 226420 366357 336484 93159 606570 474872 917921 482959 80824 206753 922706 712828 745808 992143 212093 630687 642372 447805 473989 721492 222262 232477 877012 259164 185035 429899 161274 390042 402954 834744 573934 99125 781938 434493 13439 76263 680717 552835 65896 309747 737742 492848 696696 424845 213397 184892 373037 139775 971876 956417 659755 994336 285915 555013 858485 96894 628170 928904 127335 268107 691376 544476 620538 140311 117495 646795 590149 321479 974569 111424 346496 631904 914638 765134 808440 442967 201621 921383 388305 789261 