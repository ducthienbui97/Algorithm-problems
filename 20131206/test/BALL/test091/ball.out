13662727865444
